### DISCUSSION TITLE
Hey, I have a doubt regarding managing the mdx files in development mode

### DISCUSSION THREAD

**Discussion by Gitstar-OC** [fuma-nama/fumadocs#1036] [Q&A]
Hey @fuma-nama, I am facing a problem most of the time due to large number of mdx files in my various folders. What is happening is due to the large number of mdx files and folders (currently I have 6 folders) it takes a long time to start the development mode and sometimes it takes most of my laptops available resources, so is there any way to host a separate folder at a time instead of compiling them all at once ?

**Comment by fuma-nama** [comment]
Unfortunately no, I asked the Next.js team but they currently are not working on new APIs and lazy bundling of async imports isn't supported. 

Now the only solutions are Content Collections or your custom content source using next-mdx-remote.

**Reply by Gitstar-OC** [reply]
Means what do you mean by custom content source ? Like fetching data for the mdx file using server to make it easier ?

**Reply by fuma-nama** [reply]
Custom content source is to create your own content source which allows you to define how you obtain and render content

https://fumadocs.vercel.app/docs/headless/remote
